### DISCUSSION TITLE
Table view should show errors

### DISCUSSION THREAD

**Discussion by gingerwizard** [grafana/grafana#46464] [Dashboards, Panels & Visualizations]
**Describe the feature**

Errors in sql statements are currently only shown in the chart view. These should also be shown in the table view. Typically users develop queries in the table view, to ensure the data structure is correct (or transforms are working as intended). Errors are not shown here, requiring the user to switch to the chart view to identify the issue.

**Versions**
Grafana 8.3.4

**To Reproduce**

1. Enter an invalid query in the table view
2. Switch to chart view to see error

**Expected behavior**

Errors to be shown in the table view.

**Screenshots**

Chart view

![image](https://user-images.githubusercontent.com/12695796/157661965-a9843340-68ca-4780-90e3-6d3d88d46b9f.png)

vs Table View

![image](https://user-images.githubusercontent.com/12695796/157662043-8ad48978-6bd0-4f04-83d8-4d2df4dbf382.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
